recaf-ui/src/main/java/me/coley/recaf/ui/plugin/item/RemotePluginItem.java

```diff
@@ -0,0 +1,98 @@
+package me.coley.recaf.ui.plugin.item;
+
+import dev.xdark.recaf.plugin.Plugin;
+import dev.xdark.recaf.plugin.PluginContainer;
+import dev.xdark.recaf.plugin.RecafRootPlugin;
+import dev.xdark.recaf.plugin.repository.PluginRepoItem;
+import me.coley.recaf.config.Configs;
+import me.coley.recaf.ui.plugin.DownloadFailedException;
+import me.coley.recaf.util.HttpUtil;
+import me.coley.recaf.util.Directories;
+import me.coley.recaf.util.logging.Logging;
+import org.slf4j.Logger;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Optional;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+
+/**
+ * Remote plugin information, used for UI display.
+ *
+ * @author xtherk
+ */
+public class RemotePluginItem extends PluginRepoItem {
```

Comment:
Since the super type got renamed, I may want to rename this type as well. Again, I'll nail down a consistent naming scheme with following UI integration work.